third_party/xla/third_party/tsl/tsl/platform/default/dso_loader.cc

```diff
@@ -195,19 +195,9 @@ absl::StatusOr<void*> GetHipblasltDsoHandle() {
   return GetDsoHandle("hipblaslt", "");
 }
 
-<<<<<<< HEAD
-#if (TF_ROCM_VERSION >= 60000)
-#define HIP_SO_VERSION "6"
-#else
-#define HIP_SO_VERSION "5"
-#endif
-
-absl::StatusOr<void*> GetHipDsoHandle() { return GetDsoHandle("amdhip64", HIP_SO_VERSION); }
-=======
```

Comment:
Do we have a proper fix for this now? If not we will have issues with trying to load two versions of the same library (https://github.com/ROCm/tensorflow-upstream/commit/cf09f6adea48688d1775cf3a272377239a4c5f40)

Reply:
are you referring this one? https://github.com/openxla/xla/pull/13879

Reply:
Yes, thank you! Not sure how I missed it during the last sync 😅